core/pkg/server/sender.go

```diff
@@ -1175,17 +1180,14 @@ func (s *Sender) sendExit(record *service.Record) {
 
 // sendMetric sends a metrics record to the file stream,
```

Comment:
now i'm noticing that the comment here is wrong, we upload metrics as part of the config, which is done using graphql

Reply:
good catch! I think a lot of comments in `handler.go` are also outdated, I will go through and check

Reply:
updated